cli/flox/src/commands/generations/list.rs

```diff
@@ -44,33 +47,52 @@ struct DisplayMetadata<'m> {
 }
 impl Display for DisplayMetadata<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "Description: {}", self.metadata.description)?;
-        write!(f, "Created: {}", self.metadata.created)?;
-        if let Some(last_active) = self.metadata.last_active {
-            writeln!(f)?;
-            write!(f, "Last Active: {last_active}")?;
+        let description = &self.metadata.description;
+        let created = self.metadata.created;
+        let last_active = if let Some(last_active) = self.metadata.last_active {
+            last_active.to_string()
+        } else {
+            "Now".to_string()
         };
-        Ok(())
+
+        write!(f, "{}", formatdoc! {"
+            Description: {description}
+            Created:     {created}
+            Last Active: {last_active}"})
```

Comment:
I think `Last Active` could mean the last time the environment was `flox activate`ed just as much as the last time that generation was the current generation. Struggling to come up with something better but might be worth trying to come up with something